src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $product-card-width;
```

Comment:
The variable `$product-card-width` is used for the card width. Ensure that this variable is defined elsewhere in your SCSS files, otherwise it will cause a compilation error.